superset/assets/src/visualizations/deckgl/layers/Screengrid/Screengrid.jsx

```diff
@@ -41,7 +41,7 @@ function setTooltipContent(o) {
   );
 }
 
-export function getLayer(formData, payload, onAddFilter, setTooltip, filters) {
+export function getLayer(formData, payload, onAddFilter, setTooltip, selected, onSelect, filters) {
```

Comment:
This broke the play slider (doesn't play anymore), if I revert this line back then it works